src/components/Search/Search.scss

```diff
@@ -2,47 +2,58 @@
 
 .wkp-search {
   position: absolute;
-  top: 25px;
+  top: 26px;
   z-index: 2; // To be above header div, in order to be clickable.
   right: 320px;
   left: 400px;
 
+  .wkp-search-toggle-container--open,
+  .wkp-search-toggle-container {
```

Comment:
Do you need this 2 lines ?

Reply:
You're right, they're useless. Thanx for pointing that out. Fixed and pushed